integration-tests/README.md

```diff
@@ -78,7 +78,14 @@ Basic Use Cases described above are verified in all the domain configurations. A
 | Two Operators using same target domains namespace | create chart using target domains namespace as already running first operator, verify that deployment fails with expected error |
 | Operator Helm Chart using not preexisted target domains namespace | create chart using not preexisted target domains namespace as already running first operator, verify that deployment fails with expected error |
 
+Situational Configuration Override Usecases
```

Comment:
"Configuration overrides" is the public name of this feature...  Look here: https://github.com/oracle/weblogic-kubernetes-operator/blob/master/site/config-overrides.md#overview.  We'll just references "situational configuration" as an "also known as" name, but we should just use the heading, "Configuration Overrides Use Cases."

Reply:
Done.